napari/components/_viewer/view/main.py

```diff
@@ -123,26 +126,32 @@ def screenshot(self, region=None, size=None, bgcolor=None):
         """
         return self.canvas.render(region, size, bgcolor)
 
-    def _open_images(self):
-        """Add an image layer to the viewer.
+    def _open_files(self):
+        """Add an files from the menubar.
+        """
```

Comment:
I never mention anything because I hate nitpicking (no really I do! =P), but thought I'd bring it up maybe once and for all: [PEP257](https://www.python.org/dev/peps/pep-0257/) specifies that one-liner docstrings have the closing triple-quote on the same line.

Also might as well fix the grammar, "Add an files". =P

Finally, I'm not sure I agree with the renaming to "files", because I hope to eventually add "open segmentation", "open points", etc...

Reply:
I don't mind the nitpick, i'm just sorry to create extra work for you cause I always forget! where is black up for doc strings!!!

I'll rename to `_open_images` for now as that is what it is doing. We'll have to discuss in #379 what is feasible for one "open files" function vs what need separate "open segmentation", "open points" etc. I prefer as much of the former as possible, but it depends on us figuring out what we're opening and there could be ambiguous cases